cadcdata/setup.cfg

```diff
@@ -0,0 +1,37 @@
+[build_sphinx]
+source-dir = docs
+build-dir = docs/_build
+all_files = 1
+
+[build_docs]
+source-dir = docs
+build-dir = docs/_build
+all_files = 1
+
+[upload_docs]
+upload-dir = docs/_build/html
+show-response = 1
+
+[pytest]
+minversion = 2.2
+norecursedirs = build docs/_build
+doctest_plus = enabled
+
+[ah_bootstrap]
+auto_use = True
+
+[metadata]
+package_name = cadcdata
+description = Client for accessing data at the Canadian Astronomy Data Centre
+long_description = Client for accessing data at the Canadian Astronomy Data Centre
+author = Canadian Astronomy Data Centre
+author_email = cadc@nrc-cnrc.gc.ca
+license = AGPLv3
+url = http://www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca/data
+edit_on_github = False
+github_project = opencadc/caom2tools 
+# version should be PEP386 compatible (http://www.python.org/dev/peps/pep-0386)
+version = 1.0a
+
+[entry_points]
+cadc-data = cadcdata.core:main_app
```

Comment:
That's correct. This is the preferred way of doing it. It definitely works better in testing.